rfcs/032-lineage-backend.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: lineage-backend
+- Start Date: 2021-03-31
+- RFC PR: [amundsen-io/rfcs#32](https://github.com/amundsen-io/rfcs/pull/32)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Amundsen Lineage - Backend
+
+## Summary
+
+Currently, we link to the 3rd Party lineage tool in order to show table and column lineage for tables in Amundsen. 
+The goal of this project is to have a lineage graph that is native to Amundsen so that users can find table and column lineage in one platform.
+
+## Motivation
+
+At the moment we are relying completely on any 3rd party lineage tool to show the table and column lineage in Amundsen. 
+Not everyone has access to such a 3rd party tool and want to have lineage information available natively in Amundsen.
+
+ref: https://github.com/amundsen-io/rfcs/blob/master/rfcs/025-lineage-stage-0.md
+
+## Guide-level Explanation (aka Product Details)
+
+No new concepts/definitions are being introduced as a part of this RFC. 
+
+Databuilder already has the table lineage model, which creates an upstream/downstream relation to adding to the Neo4j graph. 
```

Comment:
do you plan to change the existing model?
If so, what will be the new model interface?

The current one doesn't take into account for job/application that generate the lineage in between, do you plan to add those?

Reply:
clarified